edb/server/compiler/compiler.py

```diff
@@ -537,6 +539,70 @@ def _compile_ql_script(
 
         return sql.decode(), argmap
 
+    def _compile_query_args(
+            self, 
+            ctx: CompileContext,
+            subtypes: List[Tuple[str, Any, bool]], 
+    ) -> Tuple[bytes, uuid.UUID]:
+        # TODO(fmoor) perhaps check for zero parameters and 
+        # return a static value (b'...', uuid.UUID(b'...'))
+
+        current_tx = ctx.state.current_tx()
+        schema = current_tx.get_schema(self._std_schema)
```

Comment:
Does this look like the right schema object to be using? I am getting a few errors like this from `s_utils.typeref_to_ast()`:

```
edgedb.errors.SchemaError: cannot get item data: item 'f3000700-db1e-1114-fca2-12a8e050d706' is not present in the chema <FlatSchema gen:6104 at 0x7f5dad066490>
```

Reply:
You should be using the schema object returned with the results of the compilation of the main query (`ir.schema`)